### PR TITLE
Add tests for PythonVariableLogger

### PR DESCRIPTION
**Pull request by brsm3129**
* **Tickets addressed:** MAXGNC-1105
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

New Tests were added for PythonVariableLogger

## Verification

Tests were added

## Documentation

None

## Future work

More tests can always be used

### CODE REVIEW

File: src/tests/test_pythonVariableLogger.py
Comment (patkenneally) [inline src/tests/test_pythonVariableLogger.py]: :small_red_triangle_down: You can remove this `__main__`. The custom that this is done in other parts of Basilisk is largely now something that people do from an unknowing copy-paste, whereas it was initially done for tests that made sense to be used as example simulations. So in this case we don't need/want the main.

File: src/utilities/tests/SimpleTestModule.py
Comment (patkenneally) [inline src/utilities/tests/SimpleTestModule.py]: :small_red_triangle_down: Rather than move this, we should copy it. Tests should be as self contained as possible/reasonable. As a result, any change to this test module would affect the other test and vice-versa, making it not self contained.

**Review by patkenneally — APPROVED** [review]
Legend:
:small_red_triangle_down: *Issues* to address before merge
:large_orange_diamond: *Requests* that should not block merge, but should at least be discussed
:large_blue_circle: *Recommendations* that can be ignored if desired

File: src/tests/test_pythonVariableLogger.py
Comment (brsm3129) [reply src/tests/test_pythonVariableLogger.py]: Good point, removed

File: src/utilities/tests/SimpleTestModule.py
Comment (brsm3129) [reply src/utilities/tests/SimpleTestModule.py]: Got it that makes sense.

**Review by patkenneally — APPROVED** [review]

**Pull request merged**
